backend/settings.py

```diff
@@ -1,18 +1,19 @@
 """
 Django settings for backend project.
 
-Generated by 'django-admin startproject' using Django 5.1.4.
+Generated by 'django-admin startproject' using Django 5.2.
 
 For more information on this file, see
-https://docs.djangoproject.com/en/5.1/topics/settings/
+https://docs.djangoproject.com/en/5.2/topics/settings/
 
 For the full list of settings and their values, see
-https://docs.djangoproject.com/en/5.1/ref/settings/
+https://docs.djangoproject.com/en/5.2/ref/settings/
 """
 
 from pathlib import Path
 import os
 from django.core.management.utils import get_random_secret_key
+from csp.constants import SELF, UNSAFE_INLINE, UNSAFE_EVAL
```

Comment:
These import are not used at all in the CSP object below

Reply:
I created a PR against you change with the change. once you test, approve you can merge it here

https://github.com/PraveshKunwar/canvas-course-manager-next/pull/2